pkg/util/util.go

```diff
@@ -837,7 +837,7 @@ func IsIBMPlatform() bool {
 	isIBM := strings.HasPrefix(nodesList.Items[0].Spec.ProviderID, "ibm")
 	if isIBM {
 		// Incase of Satellite cluster is deplyed in user provided infrastructure
-		if strings.Contains(nodesList.Items[0].Spec.ProviderID, "sat-ibm") {
+		if strings.Contains(nodesList.Items[0].Spec.ProviderID, "/sat-") {
```

Comment:
Do you have a reference to the source that shows the `ProviderID` assignment in IBM cases?
Just trying to verify the change.

Reply:
It is IBM internal code. But I can share how the provider ID looks like
```
$ kubectl get nodes 172.33.16.226 -o yaml | grep -i provider
    ibm-cloud.kubernetes.io/iaas-provider: upi
          f:ibm-cloud.kubernetes.io/iaas-provider: {}
        f:providerID: {}
  providerID: ibm://62db5de4494c64ef4b5f94b057bcc557///c0hbolfw0t9eou3re3p0/sat-ip17233162-15b16ea8f23c4d44d09fb82ec08b21c6a9f85ebc
  ```